index.html

```diff
@@ -158,41 +158,51 @@ <h4>Profesional Art Printing Data More</h4>
     </div>
     <!------------------about me------------------------------------------------>
     <div class="aboutme-container">
-                <div class="aboutme-left">
-                <h4>About me</h4>
-                <p>Hey there! I can help you build a product, feature or website. Look through some of my work and
-                    experience! If you like
-                    what you see and have a project you need coded, don’t hesitate to contact me.</p>
-            </div>
-            <div class="aboutme-right">
-                <button class="gr">Get My Resume</button>
-            </div>
-
```

Comment:
- [ ] Kindly update the `font-family` on the languages to match the design template. 🙂️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/214769555-d312c7b9-56ce-4fec-9b4e-6b1531872147.png" width="200">|<img src="https://user-images.githubusercontent.com/17243325/214769557-4526ce6f-9498-4445-ab98-8059bd7550dd.png" width="200">|